### ISSUE TITLE
Obol does not check if uidNumber/gidNumber already exit

### ISSUE THREAD

**Issue by sibomarie** [clustervision/trinityX#192]
Branch: development
version: 13b3d07
environment: ha1-controller1
```
Obol can assign the same uidNumber/gidNumber to two user.
Also modify --uid  does has impact to uidNumber not uid.

```

**Comment by Marc69** [comment]
Is this only with obol user modify or also with user create? If it's only with modify I got the same issue (#197).

**Comment by hicham-a** [comment]
fixed in 3d38bb1
